src/geofabrics/processor.py

```diff
@@ -78,8 +78,8 @@ def get_lidar_file_list(self, verbose) -> list:
             self.lidar_fetcher = lidar_fetch.OpenTopography(
                 self.catchment_geometry, self.get_instruction_path('local_cache'), verbose=verbose)
             self.lidar_fetcher.run()
-            lidar_file_paths = list(pathlib.Path(self.lidar_fetcher.cache_path /
-                                    self.lidar_fetcher.dataset_prefixes[lidar_dataset_index]).glob('*.laz'))
+            lidar_file_paths = sorted(list(pathlib.Path(self.lidar_fetcher.cache_path /
+                                      self.lidar_fetcher.dataset_prefixes[lidar_dataset_index]).glob('*.laz')))
```

Comment:
Good idea to sort these but it could be even further simplified since `sorted` will create a `list` out of any `Sequence`

```suggestion
            lidar_file_paths = sorted(pathlib.Path(self.lidar_fetcher.cache_path /
                                      self.lidar_fetcher.dataset_prefixes[lidar_dataset_index]).glob('*.laz'))
```

Reply:
If you feel this reduces readability then keep it as is

Reply:
`.glob()` will produce a list so you could also do this:

```suggestion
            lidar_file_paths = pathlib.Path(self.lidar_fetcher.cache_path /
                                      self.lidar_fetcher.dataset_prefixes[lidar_dataset_index]).glob('*.laz')
           lidar_file_paths.sort()                             
```